CHANGELOG

```diff
@@ -1,5 +1,61 @@
 This file contains a summary of changes to the Oppia code base.
 
+v2.7.1 (20 December 2018)
+-------------------------
+Updates to landing pages:
+* Updates to the text on the stewards page. (#5912)
+* Add landing page for parents and teachers. (#5905)
+
+Development workflow:
+* Fix #5466: Added check to enforce that all name of controllers end with "Handler" or "Page" (#5878)
+* Fixed part of #5476: Enforce docstring arg order (#5844)
+* Fixes #5843 Changes _check_comment to check for spaces and capitals (#5852)
+* Fixes #5809: Add check to match line breaks in controller dependencies (#5833)
+* Fix docstrings tests, Swapped if blocks for _check_docstrings (#5752)
+* Remove mobile tests env var for PR builds and add a job for it (#5835)
+* Update bug_report.md (#5866)
+* Fix #5896: Set threads' daemon property to False (#5897)
+
+New Structures:
+* Add paging for questions in topic editor and story editor modifications (#5892)
+* UI modifications to story editor. (#5724)
+* Add functionality to Publish stories (#5709)
+* Revert some changes in #5810. (#5841)
+
+Improving test coverage:
+* Fixes #5861: Updated Test for Email Truncation (#5915)
+* Fix part of #5134: Added missing tests for question domain. (#5887)
+* Fixes part of #5134: Added backend tests for `core.domain.learner_progress_domain` (#5859)
+* Fix part of #5134 Backend test for core.controllers.tasks (#5840)
+* Fix #5787: Add a unit test to check blob type for uploaded audio. (#5858)
+
+Code health and documentation:
+* Added comment to Image Uploader fix (#5895)
+* Fix part of #4374: Add docstrings to python files (#5871)
+* Some follow-up fixes to style issues in #5730. (#5842)
+* Added missing docstrings in scripts/common.py and scripts/docstrings_checker.py (#5870)
+* Added missing docstrings in scripts/build.py (#5808)
+* Fix part of #4374: Add docstrings to the python files. (#5849)
+* Added docstring for clamp method and simplified usage. (#5834)
+
+Improvements to editor features
+* Fixes #3691 Implements email functionality for failed ML training job (#5902)
+* Fix #5606: Logic Proof Truncation and Display (#5864)
+* Change 'compliment' feedback to use a green background colour. (#5865)
+* Fixes #5746: Added saving log and disabled buttons while saving (#5764)
```

Comment:
Nit (optional): Maybe switch all 'Fixes' to 'Fix' in the new lines to follow Git conventions.

Reply:
Done (at all places)